Formula/apache-arrow.rb

```diff
@@ -1,10 +1,10 @@
 class ApacheArrow < Formula
   desc "Columnar in-memory analytics layer designed to accelerate big data"
   homepage "https://arrow.apache.org/"
-  url "https://downloads.apache.org/arrow/arrow-9.0.0/apache-arrow-9.0.0.tar.gz"
+  #url "https://downloads.apache.org/arrow/arrow-10.0.0/apache-arrow-10.0.0.tar.gz"
```

Comment:
probably can't do that

```suggestion
  url "https://downloads.apache.org/arrow/arrow-10.0.0/apache-arrow-10.0.0.tar.gz"
```